pkg/sippyserver/server.go

```diff
@@ -784,6 +784,18 @@ func (s *Server) jsonTestsReportFromDB(w http.ResponseWriter, req *http.Request)
 	}
 }
 
+func (s *Server) jsonTestsReportFromBigQuery(w http.ResponseWriter, req *http.Request) {
+	// Fall back to postgres if dataset is not ci_analysis_us
+	if s.bigQueryClient == nil || s.bigQueryClient.Dataset != "ci_analysis_us" {
```

Comment:
@neisw I like this redirect better. But I also have another version returning 500 error. Let me know which one you prefer.